README.md

```diff
@@ -110,36 +131,36 @@ Go back to the GitHub App and update the following settings
-| [*github_webhook_secret_key_vault_id]() | Keyvault Vault ID for GitHub App webhook secret ([data source](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/key_vault_secret)) | `string` |
-| [*github_private_key_key_vault_id]() | Keyvault Vault ID for GitHub App private key ([data source](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/key_vault_secret)) | `string` |
-| [*owners]() | The list of owners that will be assigned to all components ([data source](https://registry.terraform.io/providers/hashicorp/azuread/latest/docs/data-sources/users)) | `list(string)` |
+| <a name="azure_tenant_id"></a> [azure\_tenant\_id](#azure\_tenant\_id) | Azure tenant ID | `string` |
+| <a name="azure_subscription_id"></a> [azure\_subscription\_id](#azure\_subscription\_id) | Azure subscription ID | `string` |
+| <a name="azure_resource_group_name"></a> [azure\_resource\_group\_name](#azure\_resource\_group\_name) | Resource Group that the components and runners will be created within | `string` |
+| <a name="azure_subnet_id"></a> [azure\_subnet\_id](#azure\_subnet\_id) | Azure subnet ID | `string` |
+| <a name="name_suffix"></a> [name\_suffix](#name\_suffix) | Identifying suffix that will be appended to all components created by this module (default: `null`) | `string` |
+| <a name="github_organization"></a> [github\_organization](#github\_organization) | GitHub organization | `string` |
+| <a name="github_app_id"></a> [github\_app\_id](#github\_app\_id) | GitHub App ID | `string` |
+| <a name="github_client_id"></a> [github\_client\_id](#github\_client\_id) | GitHub Client ID | `string` |
+| <a name="github_installation_id"></a> [github\_installation\_id](#github\_installation\_id) | GitHub App installation ID | `string` |
+| <a name="azure_secrets_key_vault_resource_id"></a> [azure\_secrets\_key\_vault\_resource\_id](#azure\_secrets\_key\_vault\_resource\_id) | Key Vault ID where GitHub secrets are stored | `string` |
+| *<a name="azure_runner_default_password_key_vault_id"></a> [azure\_runner\_default\_password\_key\_vault\_id](#azure\_runner\_default\_password\_key\_vault\_id) | Key Vault ID for Azure runner default password ([data source](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/key_vault_secret))| `string` |
+| *<a name="github_client_secret_key_vault_id"></a> [github\_client\_secret\_key\_vault\_id](#github\_client\_secret\_key\_vault\_id) | Keyvault Vault ID for GitHub App client secret ([data source](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/key_vault_secret)) | `string` |
+| *<a name="github_webhook_secret_key_vault_id"></a> [github\_webhook\_secret\_key\_vault\_id](#github\_webhook\_secret\_key\_vault\_id) | Keyvault Vault ID for GitHub App webhook secret ([data source](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/key_vault_secret)) | `string` |
```

Comment:
![image](https://user-images.githubusercontent.com/75693326/216195495-4275d94d-06c4-466f-8453-b70d2c4ebc6d.png)
Well, that change made this entry render weird; maybe the variable name could be shortened, or the `*` could be moved to the data type or something?

Reply:
This is now fixed!